docs/documentation/stories/continuous-integration.md

```diff
@@ -76,28 +79,23 @@ jobs:
   build:
     working_directory: ~/my-project
     docker:
-      - image: circleci/node:8-browsers
+      # specify the version you desire here
+      # see https://hub.docker.com/r/circleci/node/tags/
+      - image: circleci/node:8.11.3-browsers
     steps:
       - checkout
-      - restore_cache:
-          key: my-project-{{ .Branch }}-{{ checksum "package.json" }}
-      - run: npm install
-      - save_cache:
-          key: my-project-{{ .Branch }}-{{ checksum "package.json" }}
-          paths:
-            - "node_modules"
-      - run: xvfb-run -a npm run test -- --no-watch --no-progress --browser=ChromeNoSandbox
-      - run: xvfb-run -a npm run e2e -- --no-progress --config=protractor-ci.conf.js
-
+      # install the latest npm to allow use of 'npm ci'
+      - run: sudo npm install -g npm@latest
+      - run: npm ci
```

Comment:
Have you checked if `npm ci` is faster than using cache on the CI? Last time I checked `npm ci` was much slower.

Reply:
Yes, you're quite right, using the cache is much faster.